### DISCUSSION TITLE
Considering using latest bedrock server  in the docker file ?

### DISCUSSION THREAD

**Discussion by q2333gh** [itzg/docker-minecraft-server#2612] [General]
its  ` https://minecraft.azureedge.net/bin-linux/bedrock-server-1.20.51.01.zip`

**Comment by itzg** [comment]
Are you aware you posted this on the Java image repo? Also, there's already code doing a dynamic lookup of the bedrock version startup. Is there a problem you're seeing or what's the context for your post?

https://github.com/itzg/docker-minecraft-bedrock-server/blob/master/bedrock-entry.sh#L13

**Reply by q2333gh** [reply]
Thanks ! I will go check out ! It helps !
